.gitignore

```diff
@@ -1,3 +1,14 @@
 # ignore python generated files 
 **/.
 **/_
+
+# OS generated files #
+######################
```

Comment:
This is a fine change, but why is your OS generating these files?

Reply:
I'm running on Mac with a Windows Bootcamp dual boot and use fat32 for my hard-drive and program with PyCharm CE. There are some weird os generated files that show up.